File: backend/src/main/java/org/jdk/project/dto/user/UserDto.java
package org.jdk.project.dto.user;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class UserDto {
  private Long id;
  private String username;
  private String nickname;
  private String email;
  private String phone;
  private String sex;
  private String avatarUrl;
  private Boolean enable;
  private String remark;
  private String createTime;
  private String updateTime;
}

File: backend/src/main/kotlin/org/jdk/project/dto/user/UserDto.kt
package org.jdk.project.dto.user

data class UserDto(
    var id: Long? = null,
    var username: String? = null,
    var nickname: String? = null,
    var email: String? = null,
    var phone: String? = null,
    var sex: String? = null,
    var avatarUrl: String? = null,
    var enable: Boolean? = null,
    var remark: String? = null,
    var createTime: String? = null,
    var updateTime: String? = null,
)
